packages/web-components/src/components/va-textarea/va-textarea.tsx

```diff
@@ -137,40 +143,91 @@ export class VaTextarea {
   }
 
   render() {
-    const { label, error, placeholder, name, required, value, hint } = this;
+    const { label, error, placeholder, name, required, value, hint, uswds } = this;
     const maxlength = this.getMaxlength();
 
-    return (
-      <Host>
-        <label htmlFor="textarea">
-          {label}
-          {required && <span class="required">{i18next.t('required')}</span>}
-        </label>
-        {hint && <span class="hint-text">{hint}</span>}
-        <span id="error-message" role="alert">
-          {error && (
-            <Fragment>
-              <span class="sr-only">{i18next.t('error')}</span> {error}
-            </Fragment>
+    if (uswds) {
+      const labelClass = classnames({
+        'usa-label': true,
+        'usa-label--error': error,
+      });
+      const inputClass = classnames({
+        'usa-textarea': true,
+        'usa-input--error': error,
+      });
+      return (
+        <Host>
+          {label && (
+            <label htmlFor="textarea" class={labelClass} part="label">
```

Comment:
In the USWDS markup, the label element has a `for` attribute set to `input-type-textarea` (it does not have `htmlFor`):

`<label ... for="input-type-textarea">`

And the `for` value should match the `id` on the textarea too:

```
<textarea ... id="input-type-textarea" ...></textarea>
```

https://designsystem.digital.gov/components/text-input/

Reply:
@jhonnyoddball `htmlFor` should be changed to `for` too:

<img width="637" alt="Screenshot 2023-03-20 at 4 42 04 PM" src="https://user-images.githubusercontent.com/872479/226472243-9d598aec-3dfe-4517-a7d5-d40059dcdf2f.png">

Reply:
I don't think we can use it. If so, we will need to update all the components to use `for`

![Screenshot 2023-03-21 at 9 43 00 AM](https://user-images.githubusercontent.com/55560129/226624502-74d6f920-cfcf-4db6-b0d6-ae40d9ef8e37.png)

Reply:
ah, I see! It converts to `for` when it compiles:
![Screenshot 2023-03-21 at 9 35 02 AM](https://user-images.githubusercontent.com/872479/226639837-1390f5c0-012a-4c71-ab73-64eea9aa5c3f.png)